Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1337,7 +1337,9 @@ def search_ip_command(ip):
 
         ip = Common.IP(
             ip=ip_address,
-            dbot_score=dbot_score
+            dbot_score=dbot_score,
+            malware_family=get_tags_for_tags_and_malware_family_fields(raw_tags, True),
+            tags=get_tags_for_tags_and_malware_family_fields(raw_tags)
```

Comment:
We are putting similar values here (and in all other places)?

Reply:
Yes, that's the request